fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -223,6 +225,12 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     @VariableMgr.VarAttr(name = ENABLE_PIPELINE, alias = ENABLE_PIPELINE_ENGINE, show = ENABLE_PIPELINE_ENGINE)
     private boolean enablePipelineEngine = true;
 
+    @VariableMgr.VarAttr(name = USE_COMPUTE_NODES)
```

Comment:
What's the usage of these two variables?

Reply:
prefer_compute_node is true means that if the cluster has compute node or this query has hdfsScanNode/icebergScanNode/HudiScanNode will use compute node to calculate except the fragment of OlapScanNode.
use_compute_nodes means that how many compute nodes this query will use.

Reply:
We should automatically decide to use how many compute nodes, needn't one session variable.

Reply:
if use don't set use_compute_nodes means use all compute node default. It's just one more option for user. Do not use it also ok for user.